agent/agent.go

```diff
@@ -55,6 +56,13 @@ type Options struct {
 	// can call Close before shutting down.
 	// Optional.
 	ShutdownCleanup bool
+
+	// SocketReuseAddrAndPort determines whether the SO_REUSEADDR and
+	// SO_REUSEADDR socket options should be set on the listening socket of
+	// the agent. This option is only effective on unix-like OSes and if
+	// Addr is set to a fixed host:port.
+	// Optional.
+	SocketReuseAddrAndPort bool
```

Comment:
Please rename this to `ReuseSocketAddrAndPort` because `SocketReuseAddrAndPort` doesn't roll off the tongue right and feels like it is exposing internal details `SO_REUSEADDR` to the user.

Reply:
Done, thanks.